contracts/test/contract.SL2RD.test.js

```diff
@@ -332,4 +464,46 @@ contract("SL2RD", (accounts) => {
         });
     }
   });
+
+  specify(
+    "Transfer next available slot to a specified address",
+    async () => {
+      const shareContract = await SHARE.deployed();
+      const splitContract = await SL2RD.new();
+      const operatorRegistry = await OperatorRegistry.deployed();
```

Comment:
It would be good to have a test case which explicitly initializes a registry using only the EOA which will call the transfer function, and validating that there's an assertion if another address attempts to make the call.